docker/app/Dockerfile

```
@@ -59,35 +91,58 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
 # Non-root user for app
 RUN useradd -m -u 10001 llamacrawl
 
-# Minimal runtime packages (psql client optional for healthchecks)
+# Minimal runtime packages + Node.js + supervisor
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)